src/legacy/core_plugins/data/public/search/search_strategy/default_search_strategy.test.ts

```diff
@@ -55,57 +50,23 @@ describe('defaultSearchStrategy', function() {
           },
         ],
         esShardTimeout: 0,
-        es: {
-          msearch: msearchMock,
+        searchService: {
           search: searchMock,
         },
       };
     });
 
-    test('does not send max_concurrent_shard_requests by default', async () => {
-      const config = getConfigStub({ 'courier:batchSearches': true });
-      await search({ ...searchArgs, config });
-      expect(searchArgs.es.msearch.mock.calls[0][0].max_concurrent_shard_requests).toBe(undefined);
-    });
-
-    test('allows configuration of max_concurrent_shard_requests', async () => {
-      const config = getConfigStub({
-        'courier:batchSearches': true,
-        'courier:maxConcurrentShardRequests': 42,
-      });
-      await search({ ...searchArgs, config });
-      expect(searchArgs.es.msearch.mock.calls[0][0].max_concurrent_shard_requests).toBe(42);
-    });
-
-    test('should set rest_total_hits_as_int to true on a request', async () => {
```

Comment:
Actually these are being injected here: https://github.com/elastic/kibana/blob/ecab2073ae8a730d9913d4b66c0e3f5181b1d92c/src/plugins/data/server/search/es_search/es_search_strategy.ts#L34-L36

I believe we can get rid of the `max_concurrent_shard_requests` tests because they are only related to `_msearch` calls.

Reply:
Actually, `ignore_throttled` is not being set yet, but needs to be.

Reply:
Restore tests